Clarify built-in `blake2b` output size in docs

ross-weir opened the pull request.

polarker commented on protocol/src/main/resources/ralph-built-in-functions.json:
```diff
@@ -887,11 +887,11 @@
     "category": "Cryptography",
     "signature": "fn blake2b!(data:ByteVec) -> (ByteVec)",
-    "doc": "Computes the Blake2b hash of the input.",
+    "doc": "Computes the Blake2b256 hash of the input.",
```
Nit: Let's use the `Blake2b-256`, which seems a more common naming pattern.

polarker commented on protocol/src/main/resources/ralph-built-in-functions.json:
```diff
@@ -887,11 +887,11 @@
       "@param data the input data to be hashed"
     ],
-    "returns": "@returns the hash result"
+    "returns": "@returns the 32 byte hash result"
```
Nit: 32 byte -> 32 bytes

ross-weir commented:
> @ross-weir Thanks for the update.
> 
> You will also need to run `make update-builtin-doc` to update the generated docs for the built-in functions.
> 
> The generated doc will be used by the doc website and the vscode Ralph extension

Ops, I edited the generated json directly originally. Updated codegen instead

Also addressed your comments
